### PR TITLE
Deploying to Heroku

### PR DESCRIPTION
**Pull request by ckiteou**
Created a welcome message on root
Ready to deploy to Heroku

### CODE REVIEW

**Comment by dbugsy** [comment]
:shipit: :shipit:

File: spec/helpers/welcome_helper_spec.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+# Specs in this file have access to a helper object that includes
+# the WelcomeHelper. For example:
+#
+# describe WelcomeHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
```
Comment (dbugsy) [inline spec/helpers/welcome_helper_spec.rb]: remove commented code

File: spec/views/welcome/index.html.erb_spec.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe "welcome/index.html.erb", type: :view do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (dbugsy) [inline spec/views/welcome/index.html.erb_spec.rb]: remove auto generated stuff if you do't plan to use it.

**Review by dbugsy — APPROVED** [review]
2 small changes, then all good.

**Comment by congcongbo** [comment]
:shipit:

**Comment by BDCraven** [comment]
:shipit:

**Comment by dbugsy** [comment]
That's merged - please ensure that you proof it visually and then move the ticket to 'done'

**Comment by ckiteou** [comment]
Visually proofed and ticket moved to 'done'.

**Pull request merged**